src/components/Combobox/__snapshots__/Combobox.test.tsx.snap

```diff
@@ -0,0 +1,254 @@
+}
+
+.pounce-1 {
+  pointer-events: none;
+  font-size: 0.875rem;
+  padding-left: 16px;
+  padding-right: 16px;
+  color: #bdbdbd;
+  top: 0;
+  left: 0;
+  position: absolute;
+  transform-origin: center left;
+  -webkit-transform: translate(0, 14px) scale(1);
+  -moz-transform: translate(0, 14px) scale(1);
+  -ms-transform: translate(0, 14px) scale(1);
+  transform: translate(0, 14px) scale(1);
+  -webkit-transition: -webkit-transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
+  transition: transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
+  font-weight: 400;
+}
+
+.pounce-5 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  -webkit-justify-content: center;
+  justify-content: center;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  opacity: 1;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 8px;
+  pointer-events: none;
+}
+
+.pounce-4 {
+  outline: none;
+  line-height: 1;
+  border-radius: 4px;
+  height: 32px;
+  width: 32px;
+  cursor: pointer;
+  color: #f6f6f6;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  background-color: transparent;
+  -webkit-transition: all 0.1s linear;
+  transition: all 0.1s linear;
+}
+
+.pounce-4:focus {
+  background-color: rgba(255,255,255,0.1);
+}
+
+.pounce-3 {
+  display: inline-block;
+  vertical-align: sub;
+  -webkit-flex-shrink: 0;
+  -ms-flex-negative: 0;
+  flex-shrink: 0;
+  width: 20px;
+  height: 20px;
+  color: currentColor;
+}
+
+
+
+<div>
+  <div
+    aria-expanded="false"
+    aria-haspopup="listbox"
+    aria-labelledby="downshift-0-label"
+    class="pounce-6"
+    role="combobox"
+  >
+    <div
+      class="pounce-6"
+    >
+      <div
+        aria-disabled="false"
+        class="pounce-2"
+      >
+        <input
+          aria-autocomplete="list"
+          aria-disabled="false"
+          aria-invalid="false"
+          aria-labelledby="downshift-0-label"
+          aria-readonly="true"
+          autocomplete="off"
+          class="pounce-0"
+          id="downshift-0-input"
+          placeholder="Select manufacturer"
+          readonly=""
+          type="text"
+          value=""
+        />
+        <label
+          class="pounce-1"
+          for="downshift-0-input"
+          id="downshift-0-label"
+        >
+          Choose a car manufacturer
+        </label>
+      </div>
+      <div
+        class="pounce-5"
+      >
+        <button
+          aria-label="Toggle Menu"
+          aria-pressed="false"
+          class="pounce-4"
+          tabindex="-1"
+          type="button"
+        >
+          <svg
+            class="pounce-3"
+            role="presentation"
+            viewBox="0 0 24 24"
+          >
+            <path
+              d="M7 10L12 15L17 10H7Z"
+            />
+          </svg>
+        </button>
+      </div>
+    </div>
+    <ul
```

Comment:
I would suggest firing some results rendered otherwise the snapshot is a bit off.

Reply:
ah right! I'll update it.